web/package.json

```diff
@@ -2,71 +2,64 @@
   "name": "docat-web",
   "version": "0.1.0",
   "private": true,
-  "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "test:unit": "vue-cli-service test:unit",
-    "lint": "vue-cli-service lint"
-  },
   "dependencies": {
-    "babel-runtime": "^6.26.0",
-    "core-js": "^3.26.0",
-    "html-loader": "^0.5.5",
-    "node-sass": "^6.0.0",
-    "sass-loader": "^10.0.5",
-    "semver": "^7.3.7",
-    "v-tooltip": "^2.1.3",
-    "vue": "^2.6.10",
-    "vue-markdown": "^2.2.4",
-    "vuelidate": "^0.7.4"
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
+    "@mui/icons-material": "^5.10.9",
+    "@mui/material": "^5.10.11",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^13.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/jest": "^27.0.1",
+    "@types/lodash": "^4.14.191",
+    "@types/node": "^16.7.13",
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "eslint": "^8.0.1",
+    "http-proxy-middleware": "^2.0.6",
+    "lodash": "^4.17.21",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-markdown": "^8.0.3",
+    "react-router-dom": "^6.4.2",
+    "react-scripts": "5.0.1",
+    "react-tooltip": "^4.4.3",
+    "typescript": "*",
+    "web-vitals": "^2.1.0"
   },
-  "devDependencies": {
-    "@vue/cli-plugin-babel": "^5.0.8",
-    "@vue/cli-plugin-eslint": "^4.0.0",
-    "@vue/cli-plugin-unit-jest": "~4.2.0",
-    "@vue/cli-service": "^4.0.0",
-    "@vue/test-utils": "^1.3.0",
-    "babel-eslint": "^10.0.3",
-    "eslint": "^5.16.0",
-    "eslint-plugin-vue": "^6.2.2",
-    "vue-material": "^1.0.0-beta-11",
-    "vue-router": "^3.1.3",
-    "vue-template-compiler": "^2.6.10"
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --watchAll=false --testMatch **/src/**/*.test.ts --passWithNoTests",
+    "jest": "react-scripts test --watchAll=false --testMatch **/src/**/*.test.ts --passWithNoTests",
```

Comment:
Would remove this as 'test' should be enough i guess?

Reply:
I think the pipline uses `npm jest` that's why

Reply:
I would suggest to change this in the pipeline and remove the 'jest' command

Reply:
Sounds reasonable :+1: